release-notes/opensearch-release-notes-2.0.0.md

```diff
@@ -0,0 +1,637 @@
+# OpenSearch and Dashboards 2.0.0 Release Notes
+
+## Release Highlights
+
+* Document level alerting allows users to create monitors that can generate alerts per document.
+* Lucene 9 is now used in OpenSearch
+* The Geo Map Tiles in OpenSearch Dashboards are updated and now have a pipeline to update them more frequently.
+* Document level security now supports term lookup queries.
+
+### OpenSearch Notifications
+* OpenSearch 2.0.0 is the first official release with OpenSearch Notifications.
+* Notifications consist of three plugins, `notifications-core` and `notifications` backend plugins for OpenSearch, and a `notificationsDashboards` frontend plugin for OpenSearch Dashboards.
+
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 2.0.0 include the following features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/2.0/release-notes/opensearch.release-notes-2.0.0.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.0/release-notes/opensearch-dashboards.release-notes-2.0.0.md).
+
+## FEATURES
+
+### OpenSearch Index Management
+* Adds shrink action to ISM ([#326](https://github.com/opensearch-project/index-management/pull/326))
+* Notification integration with IM ([#338](https://github.com/opensearch-project/index-management/pull/338))
+
+
+### OpenSearch Index Management Dashboards Plugin
+* Adds UI for shrink action ([#176](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/176))
+
+
+### OpenSearch Job Scheduler
+* Enables locking on an arbitrary lockID ([164](https://github.com/opensearch-project/job-scheduler/pull/164))
+
+
+### OpenSearch Notifications
+* Use Notifications to manage your notification channel configurations in a centralized location and send messages to these channels.
+* This release also adds notification-specific REST APIs for CRUD operations on channels and internal transport APIs for integrating/communicating with other plugins.
+
+
+### OpenSearch Performance Analyzer
+* Adds setting to enable/disable Thread Contention Monitoring ([#171](https://github.com/opensearch-project/performance-analyzer/pull/171))
+* Fixes calculation of average thread blocked time and average thread waited time ([#118](https://github.com/opensearch-project/performance-analyzer-rca/pull/118))
```

Comment:
Since this bullet point is a fix to an existing feature, I would recommend putting it in as an enhancement.